docs/migration/incidents/gas-miscalculation.md

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_label: 'Gas Miscalculation'
```

Comment:
This issue was not about gas miscalculation.

It was about an error in rounding the number of LYXe when clicking the unit button.

The gas issue was a side effect of MetaMask not being able to predict the correct gas because of an obviously failing tx.

I think this page is not relevant. It adds a lot of content for low value. The text here

https://support.lukso.network/migration/introduction#im-getting-very-high-gas-fees-when-using-metamask-whats-going-on

Explains it correctly.

It is not as important as the system being down. It imo creates noise. We don't need that I think.